src/scss/pages/_repo_detail.scss

```diff
@@ -0,0 +1,22 @@
+.detail-container {
```

Comment:
Three things:
1. Please use the $color- values defined in _colors.scss, and use `em` for padding, etc - this keeps us pretty easily 'reconfigurable'.
2. You can define a `h3` inside your class, but please not at top-level, otherwise it effects everyone else.
3. You will need to add `@import 'pages/repo_view';` to `main.scss` so that it gets compiled into the site css.